src/pages/User/components/PullRequests/index.js

```diff
@@ -1,14 +1,35 @@
 import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
-
-import { GITHUB_TOKEN, TOTAL_PR_COUNT, TOTAL_OTHER_PR_COUNT } from '../../../../config';
 import ShareButtons from './ShareButtons';
 import LoadingIcon from './LoadingIcon';
 import PullRequest from './PullRequest';
 import IssuesLink from './IssuesLink';
 import MeLinkInfo from './MeLinkInfo';
+import NotAMember from '../Modal/NotAMember';
 import ErrorText from './ErrorText';
 import UserInfo from './UserInfo';
+import fetchInfoFromGitHub from '../../../../utils/fetchInfoFromGitHub';
```

Comment:
This could have been shortened out to ‘utils/....’ by using NODE_PATH=src in .env file. Optional.